### PR TITLE
feat: deprecate learner view code

### PR DESCRIPTION
**Pull request by alangsto**
Remove all code related to learner specific data. This includes

- any code that involved elastic search
- learner view endpoints
- learner serializers
- learner view tests

### CODE REVIEW

File: analytics_data_api/v0/apps.py [hunk 1]
```diff
@@ -1,28 +1,6 @@
 from django.apps import AppConfig
-from django.conf import settings
-from elasticsearch_dsl import connections
```
Comment (ashultz0) [inline analytics_data_api/v0/apps.py:3]: can we strip elasticsearch support out of requirements?

File: analytics_data_api/middleware.py [hunk 2]
```diff
@@ -134,24 +95,6 @@ def status_code(self):
         return status.HTTP_400_BAD_REQUEST
 
 
-class ParameterValueErrorMiddleware(BaseProcessErrorMiddleware):
```
Comment (ashultz0) [inline analytics_data_api/middleware.py:137]: I like how you've found and smashed these associated classes

File: analyticsdataserver/settings/base.py [hunk 3]
```diff
@@ -90,29 +90,6 @@
...
-ELASTICSEARCH_CONNECTION_CLASS = None
-# only needed with BotoHttpConnection, e.g. 'us-east-1'
-ELASTICSEARCH_CONNECTION_DEFAULT_REGION = None
-
-DEFAULT_ELASTICSEARCH_INDEX_SETTINGS = {
-    'number_of_shards': 1,
-    'number_of_replicas': 0
-}
-ELASTICSEARCH_INDEX_SETTINGS = environ.get('ELASTICSEARCH_INDEX_SETTINGS', DEFAULT_ELASTICSEARCH_INDEX_SETTINGS)
-########## END ELASTICSEARCH CONFIGURATION
```
Comment (ashultz0) [inline analyticsdataserver/settings/base.py:114]: so satisfying

**Comment by ashultz0** [comment]
I'm pretty heartened that you were able to get rid of the elasticsearch stuff even without removing module engagement

File: analytics_data_api/v0/apps.py (hunk 1 above)
Comment (alangsto) [reply analytics_data_api/v0/apps.py:3]: Just ripped it out!

**Review by ashultz0 — APPROVED** [review]

**Pull request merged**